Karen.java

```diff
@@ -0,0 +1,63 @@
+import com.sun.deploy.util.StringUtils;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Karen Xiao
+ * @see 5th Anniversary Class Report
+ * @since 11/9/18
+ */
+
+public class Karen
+{
+    private Integer age;
+    private String currentCity;
+    private String job;
+    private String company;
+    private static List<String> freeTime = Arrays.asList("Travel", "Dance", "Eat Hot Pot", "Paint", "Run", "Climb");
```

Comment:
should be a final constant. I'd lift it to the top of the class and capitalize it: `private static final List<String> FREE_TIME`